cloud/store/raft.go

```diff
@@ -83,12 +85,23 @@ func (st *Store) Open() (err error) {
 	}
 	log.Printf("raft.NewTCPTransport  address=%v tcpAddress=%v maxPool=%v timeOut=%v\n", address, tcpAddr, tcpMaxPool, tcpTimeout)
 
+	rLog := rLog{logStore}
```

Comment:
this will trigger the linter, we shouldn't use anonym field order when doing in-line initialization.